CleverSDK/Classes/CLVOAuthManager.m

```diff
@@ -111,18 +179,27 @@ + (BOOL)handleURL:(NSURL *)url sourceApplication:(NSString *)sourceApplication a
         kvpairs[kv[0]] = kv[1];
     }
 
+    // if code is missing, then this is a Clever Portal initiated login, and we should kick off the Oauth flow
     NSString *code = kvpairs[@"code"];
-
     if (!code) {
+        CLVOAuthManager* manager = [self sharedManager];
+        if (manager.alreadyMissedCode) {
```

Comment:
Just want to double check this -- does a value of `NO` evaluate to false?

Reply:
correct! [NO is defined as 0](https://developer.apple.com/documentation/objectivec/no?language=objc)